src/routes/event-match.tsx

```diff
@@ -172,6 +172,9 @@ const EventMatch = ({ eventKey, matchKey }: Props) => {
               class={css`
                 overflow-y: hidden;
                 grid-area: analysisTable;
+                @media screen and (width > 118.75rem) {
```

Comment:
Can you do `max-width: 100%` to make it use the scrollbar instead of using a media query? The analysis table does not have a fixed width and it changes year-to-year and also a little bit based on font rendering, etc.